src/utils/theme/index.js

```diff
@@ -28,6 +28,14 @@ export const COLORS = {
   whiteSmoke: '#F4F4F4',
   bluePrimary: '#4782D7',
 
+  anonPmGroupChannel: '#11333B',
+  anonTopicChannel: '#184A57',
+  anonPostNotificationChannel: '#17272B',
+
+  signedPmGroupChannel: '#112A3D',
+  signedTopicChannel: '#1D3E5E',
+  signedPostNotificationChannel: '#16202A',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of these new color constants is a good way to manage the different background colors for each channel type. However, it might be more maintainable and scalable if you use a more structured approach. Consider creating an object or a map that associates channel types with their respective colors. This way, when a new channel type is added, you only need to update this map instead of adding a new constant and modifying the logic where these constants are used.

Here's an example of what this could look like:

```javascript
const channelColors = {
  anonPmGroupChannel: '#11333B',
  anonTopicChannel: '#184A57',
  anonPostNotificationChannel: '#17272B',
  signedPmGroupChannel: '#112A3D',
  signedTopicChannel: '#1D3E5E',
  signedPostNotificationChannel: '#16202A',
};
```

Then, in your components, you can simply access the color like so: `channelColors[channelType]`. This will return the color associated with the given `channelType`, or `undefined` if the `channelType` does not exist in the map.


<!-- This is an auto-generated comment by OSS CodeRabbit -->